src/python/grpcio/grpc/__init__.py

```diff
@@ -1219,12 +1219,15 @@ def server(thread_pool, handlers=None):
       only handlers the server will use to service RPCs; other handlers may
       later be added by calling add_generic_rpc_handlers any time before the
       returned Server is started.
+    options: A sequence of string-value pairs according to which to configure
+      the created server
 
   Returns:
     A Server with which RPCs can be serviced.
   """
   from grpc import _server
-  return _server.Server(thread_pool, () if handlers is None else handlers)
+  return _server.Server(thread_pool, () if handlers is None else handlers,
+                        options)
```

Comment:
`None` is there in the API for developer-user convenience but we shouldn't pass it around within our own code since it doesn't make sense as its own thing distinct from an empty sequence. Handle the developer-user's passing `None` here rather than any further into our system.